srv/admin-service.cds

```diff
@@ -32,6 +32,12 @@ annotate AdminService.Books with @cds.search : {
   title
 };
 
+annotate AdminService.OrderItems with  {
+  @mandatory
+  @assert.range: [(0), _]
+  quantity
+};
```

Comment:
Should we do this on db-level already? Most other validations like these seem to be defined there.